### PR TITLE
Port to Vivado 2018.3

### PR DESCRIPTION
**Pull request by arpansur**
Modified the script to make it compatible for both 2017.2 and 2018.3

### CODE REVIEW

File: fpga/bigpulp-z-70xx/tcl/ps7_bd.tcl [hunk 1]
```diff
@@ -353,10 +353,11 @@ if { $::env(RAB_AX_LOG_EN) } {
     set rab_ar_bram_ctrl_host \
         [ create_bd_cell \
             -type ip \
-            -vlnv xilinx.com:ip:axi_bram_ctrl:4.0 \
+            -vlnv xilinx.com:ip:axi_bram_ctrl:4.1 \
```
Comment (andreaskurth) [inline fpga/bigpulp-z-70xx/tcl/ps7_bd.tcl]: I think there is a commit missing to keep supporting Vivado 2017.2.  Did you commit and push those changes?

File: fpga/bigpulp/tcl/ic_wrapper_bd.tcl [hunk 2]
```diff
@@ -7,13 +7,20 @@ if { [version -short] == "2016.3" } {
   set XLCONCAT_VERSION "2.1"
   set BLK_MEM_GEN_VERSION "8.3"
   set MB_VERSION "10.0"
-} else { 
+} elseif {
```
Comment (andreaskurth) [inline fpga/bigpulp/tcl/ic_wrapper_bd.tcl]: The condition is missing here.

**Review by andreaskurth — CHANGES_REQUESTED** [review]
Hi Arpan, thanks for your contribution! Please see my two comments.

File: fpga/bigpulp-z-70xx/tcl/ps7_bd.tcl (hunk 1 above)
Comment (arpansur) [reply fpga/bigpulp-z-70xx/tcl/ps7_bd.tcl]: May be I did some mistake, that's why it took the old file. Sorry for that. I added the file again. Please let me know if you got the request for approval.

File: fpga/bigpulp/tcl/ic_wrapper_bd.tcl (hunk 2 above)
Comment (arpansur) [reply fpga/bigpulp/tcl/ic_wrapper_bd.tcl]: I could not see the usage of the file in the pulp design I ran. Is it used for some different design?

**Review by andreaskurth — APPROVED** [review]

**Pull request merged**